packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -52,15 +40,9 @@ const propTypes = {
    * An ISO 8601 string representation of the initial default date to show in the date input. This prop name is derived from react-datepicker but is analogous to defaultValue for a form input field.
    */
   selectedDate: PropTypes.string,
-  /**
-   * An ISO 8601 string representation of the start date for a date range.
-   */
-  startDate: PropTypes.string,
 };
 
 const defaultProps = {
```

Comment:
Looks like `endDate` and `startDate` defaults were added in the [PR](https://github.com/cerner/terra-core/pull/452/files#diff-06f07d3cca7012107347dc63be00c147R65) you listed.